tests/test_cloudevent_functions.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import pathlib
+import pytest
+from cloudevents.sdk import marshaller
+from cloudevents.sdk.event import v1
+from cloudevents.sdk.event import v03
+from cloudevents.sdk import converters
```

Comment:
```suggestion
import cloudevents.sdk
import cloudevents.sdk.event
```

Generally preferable to use the full import here instead, to make it more clear what is coming from the 3rd party library and what is internal.